### ISSUE TITLE
Make it clear what swipe gestures do

### ISSUE THREAD

**Issue by bubelov** [bubelov/news#10]

**Comment by DanielSiepmann** [comment]
I'm using Android 9 on an Samsung and still don't have labels or info for swipe actions.
I've the recent version 0.2.4 of your app which includes https://github.com/bubelov/news/commit/dfa1b297d7d2e0b321a65f20de104b71b7bf8398.

Looks like "snackbar" does not work for whatever reason. I'm not a kotlin nor mobile dev, just a web dev, so I don't know how I might help you here :) But I would like to have some hint about the action, so I don't have to remember which direction has which effect.

**Comment by bubelov** [comment]
@DanielSiepmann are you sure it's 0.2.4? I released it 10 days ago but it seems like F-Droid didn't pick it up yet. You can try to download the dev APK directly from GitHub release page. It's not as fast as production builds but it works fine on my $95 Samsung =)

**Comment by DanielSiepmann** [comment]
Thanks for responding.

At least f-droid tells I've installed 0.2.3 and that it was added 09.01.21.
The OS settings "App" entry states version 0.2.3 for the app.

As written before, I'm not an Android dev. Maybe the snackbar is not available for my Android version, or something else?

I'll try the GitHub APK.

I also added Screenshots (Sorry for German language)
![image](https://user-images.githubusercontent.com/354250/105832167-e597e600-5fc7-11eb-83b5-315c7bce0256.png)
![image](https://user-images.githubusercontent.com/354250/105832193-ee88b780-5fc7-11eb-97ac-52d0e2de7df0.png)
![image](https://user-images.githubusercontent.com/354250/105832179-e9c40380-5fc7-11eb-896f-7b5cb30b46c5.png)
